rust-toolchain

```diff
@@ -1 +1 @@
-1.54
\ No newline at end of file
+nightly
```

Comment:
This should be a stable version. This file applies the version to go-cosmwasm (the rust part), sgx-vm, and the example contracts, all of which do not require nightly.